### PR TITLE
OSDOCS-10444: updated the offline deployments links in relnotes Micros…

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.15

Issue:
[OSDOCS-10444](https://issues.redhat.com/browse/OSDOCS-10444)

Link to docs preview:
[Embedding MicroShift containers for offline deployments](https://75464--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes.html#microshift-4-15-0-dp)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -199,7 +199,7 @@ Issued: 2024-02-27
 
 {product-title} release 4.15.0 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:7200[RHSA-2023:7200] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7198[RHSA-2023:7198] advisory.
 
-For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```
Comment (mburke5678) [inline microshift_release_notes/microshift-4-15-release-notes.adoc:202]: I know this is outside of the scope of this PR, we have a [new format for external links](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute). Don't know if you want to updates the links in this PR. 

```suggestion
For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments] (Microsoft documentation).
```

Comment (aravipra) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:202]: @mburke5678 These are internal links. xref points to the MicroShift docs.

**Comment by aravipra** [comment]
@mburke5678 awaiting LGTM to proceed

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Pull request merged**